### PR TITLE
Doc/translation pt br

### PR DESCRIPTION
**Pull request by barreto**

### CODE REVIEW

File: README-PTBR.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+1. Selecione dois numeros primos grandes, x e y. Os números primos precisam ser grandes para que seja difícil para alguém decifrá-los.
+2. Calcule n = x * y
+3. Calcule a função totiet: ϕ(n) = (x−1)*(y−1)
+4. Selecione um inteiro *e*, de tal forma que *e* seja co-primo de ϕ(n) e 1 < *e* < ϕ(n). O par de números (n,*e*) constitui a chave pública..
+> Dois inteiros são primos se o único inteiro positivo que os divide for 1.
+
+5. Calcule `d` de modo que `e.d = 1 mod ϕ(n)`
+
+**TRANSLATE HERE: Add can be found using the extended euclidean algorithm. The pair (n,d) makes up the private key.**
+
```
Comment (barreto) [inline README-PTBR.md]: Sorry @thalees , what did you mean with "Add can be found using the extended euclidean algorithm. The pair (n,d) makes up the private key"?

**Review by barreto — COMMENTED** [review]
Sorry @thalees  , what did you mean with "Add can be found using the extended euclidean algorithm. The pair (n,d) makes up the private key"?

File: README-PTBR.md (hunk 1 above)
Comment (thalees) [reply README-PTBR.md]: Hummm, my bad 😅 

The correct sentence would be _" `d` can be found... "_

Could you adjust the README in English, please?

Comment (barreto) [reply README-PTBR.md]: No problem, I'll fix this with one more commit.

**Pull request merged**